etl/storage/storage_test.go

```diff
@@ -240,6 +241,50 @@ func TestStorage_GetJetDrops(t *testing.T) {
 	require.Contains(t, expected, jetDrops[1])
 }
 
+func TestStorage_GetJetDropsWithParams(t *testing.T) {
+	defer testutils.TruncateTables(t, testDB, []interface{}{models.Record{}, models.JetDrop{}, models.Pulse{}})
+	s := NewStorage(testDB)
+
+	firstPulse, err := testutils.InitPulseDB()
+	assert.NoError(t, err)
+	err = testutils.CreatePulse(testDB, firstPulse)
+	assert.NoError(t, err)
+	jetDropForFirstPulse1 := testutils.InitJetDropDB(firstPulse)
+	err = testutils.CreateJetDrop(testDB, jetDropForFirstPulse1)
+	assert.NoError(t, err)
+	jetDropForFirstPulse2 := testutils.InitJetDropDB(firstPulse)
+	err = testutils.CreateJetDrop(testDB, jetDropForFirstPulse2)
+	assert.NoError(t, err)
+
+	secondPulse, err := testutils.InitPulseDB()
+	assert.NoError(t, err)
+	err = testutils.CreatePulse(testDB, secondPulse)
+	assert.NoError(t, err)
+	jetDropForSecondPulse := testutils.InitJetDropDB(secondPulse)
+	err = testutils.CreateJetDrop(testDB, jetDropForSecondPulse)
+	assert.NoError(t, err)
+
+	jetDrops, total, err := s.GetJetDropsWithParams(firstPulse, nil, 20, 0)
```

Comment:
Why several tests in one? if you want to create test data only once, here is perfect place to use `t.Run`

Reply:
replaced